app/styles/_library/components/_whats-on.scss

```diff
@@ -1,84 +1,93 @@
 @import 'episode';
 @import 'schedule';
 
-.whats-on {
-  padding: 20px;
-  max-width: 728px;
+$live-indicator-dot-color: #de1e3d;
```

Comment:
I think we should stay consistent with using css properties for variables. looks like this can be local within `.whats-on`. Is it used anywhere else?

Reply:
i'll update the vue version now so it uses a css variable